src/static/js/ours/client.js

```diff
@@ -77,10 +77,15 @@ CODALAB.api = {
     re_run_phase_submissions: function (phase_pk) {
         return CODALAB.api.request('GET', `${URLS.API}phases/${phase_pk}/rerun_submissions/`)
     },
+    get_front_page_competitions: function (data) {
+        return CODALAB.api.request('GET', URLS.API + "front_page_competitions/", data)
+    },
+    by_the_numbers: function (data) {
+        return CODALAB.api.request('GET', URLS.API + "by_the_numbers", data)
```

Comment:
trailing `/` on url.
can also use formatted strings instead of string concatenation, so we are consistent, even though we totally aren't haha.
```javascript
`${URLS.API}by_the_numbers/`
```

Reply:
Trailing slash added to api request.